vendor/engines/split_accounts/app/services/split_accounts/split_account_builder.rb

```diff
@@ -27,6 +27,7 @@ def after_build
     # Make sure this happens after the splits are built.
     def set_expires_at
       account.expires_at = account.earliest_expiring_subaccount.try(:expires_at)
+      account.expires_at ||= Time.current
```

Comment:
Maybe a comment that this should only happen in an invalid creation state where all subaccounts are blank, and that it's only there to suppress an error message?